### DISCUSSION TITLE
Jekyll_app1htmlFile/jekyll/onefileapp/2018/12/17/Kategorie_aplikacji

### DISCUSSION THREAD

**Discussion by giscus** [andrzejQ/Jekyll_app1htmlFile#16] [Komentarze]
# Jekyll_app1htmlFile/jekyll/onefileapp/2018/12/17/Kategorie_aplikacji

Moje symbole kategorii 1-plikowych aplikacji: (o1) - 1 plik off-line; (i1) 1 plik i otwarte zasoby internetowe; (x1) - “prawie” 1 plik + kilka statycznych plików; (b1) skryptozakładka

http://localhost:4000/Jekyll_app1htmlFile/jekyll/onefileapp/2018/12/17/Kategorie_aplikacji.html

**Comment by andrzejQ** [comment]
Nowa wersja komentarzy z użyciem `giscus -> GitHub Discussions`.
